README.md

```diff
@@ -0,0 +1,125 @@
+# Authentication with Devise
+
+## Learning Objectives
+
+  1. Describe the major architecture and modules of Devise.
+
+## Overview
+
+[Devise] is a gem for when you have a lot of needs.
```

Comment:
Maybe "authentication needs?" I know it's in the the title but on first pass it felt like there was a word missing.